apps/entity-service/src/entities/dto/media-association.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { Media } from "@terramatch-microservices/database/entities";
+import { AssociationDto, AssociationDtoAdditionalProps } from "./association.dto";
+import { UserDto } from "@terramatch-microservices/common/dto/user.dto";
+import { pickApiProperties } from "@terramatch-microservices/common/dto/json-api-attributes";
+export interface MediaAssociationDtoAdditionalProps extends AssociationDtoAdditionalProps {
```

Comment:
Please keep empty newlines between logical blocks to make it easier to visually parse a TS file. In general, I keep an empty line above all class, type (and interface), and method declarations.

Additionally, if the `AssociationDtoAdditionalProps` was moved to a interface just to make this `extends` possible, then please revert that and use the intersection operator instead:
```
export type MediaAssociationDtoAdditionalProps = AssociationDtoAdditionalProps & { 
```

Reply:
got it.